Pod/Classes/Cards/CardTypeRegister.swift

```diff
@@ -8,12 +8,32 @@
 
 import UIKit
 
+/// A `CardTypeRegister` is used to maintain the range of accepted card types. You can provide different card type registers for different CardTextField's and customize the range of accepted card types individually.
 public class CardTypeRegister {
     
+    /**
+     The default card type register, shared among all CardTextFields. Initially, this card type register contains:
+     - Caishen.AmericanExpress
+     - Caishen.DinersClub
+     - Caishen.Discover
+     - Caishen.JCB
+     - Caishen.MasterCard
+     - Caishen.Visa
+     */
     public static let sharedCardTypeRegister = CardTypeRegister()
     
+    /// An array of all registered card types. You can edit this array with `registerCardType`, `unregisterCardType` or `setRegisteredCardTypes`.
     public private(set) var registeredCardTypes: [CardType]
     
+    /**
+     Creates a new `CardTypeRegister` with the following card types:
+     - AmericanExpress
```

Comment:
This needs to be consistent with above; either include the namespace or disregard it. That said, I'm ok with not including this in either place but instead within the default constructor of CardTypeRegister ... again, for maintenance reasons.